tests/test_cram.py

```diff
@@ -242,6 +242,15 @@ def test_decode_itf8_array(self):
             self.assertEqual(results, [1, 128, 268435456, 2 ** 32 - 1])
 
     def test_encode_decode_itf8(self):
+        """
+        Tests ITF-8 encoding and decoding functions.
+
+        Encodes and decodes all binary numbers: 0, 1, 2**1, 2**2, 2**3, ... , up until the highest 32-bit unsigned
```

Comment:
Does this encode/decode binary numbers in general, or just powers of 2?

Reply:
The functions work on all numbers in the range, which for this is up to 2**64 - 1.  The test is testing all binary numbers, plus the numbers to either side (`2 ** n + 1` and `2 ** n - 1` respectively).

Reply:
The docstring should be updated with this comment. In a new PR I guess.